microsoft-edge/webview2/release-notes.md

```diff
@@ -19,6 +19,65 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime or SDK specific.  
 
+## 1.0.955-prerelease
+
+Release Date: July 26, 2021  
+
+[NuGet package][NuGetGallery1.0.955-prerelease] \| Minimum Microsoft Edge version to load: 86.0.616.0 or newer \| Full API Compatibility: 93.0.955.0 or newer  
+
+### General  
+*   General WebView2 performance improvements.
+*   Added some ETW tracing support for WebView2.
+*   Removed Microsoft branding from edge://history.
+*   WebView2 has a new default Download UI.
+
+#### Experimental Features  
```

Comment:
One more feature I collected is Disable PDF Tool Bar(https://microsoft.visualstudio.com/DefaultCollection/Edge/_git/chromium.src/pullrequest/6255496?path=%2Fedge_embedded_browser%2Fclient%2Fwin%2Fcurrent%2Fwebview2staging.idl), it that expected to not be included here? I'm not sure, just confirm.

Reply:
@jasonstephen15 here's the link
```
[Webview2ReferenceWin32Icorewebview2experimentalsettings6ViewWebview210955PrereleaseGetHiddenpdftoolbaritems]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalsettings6?view=webview2-1.0.955-prerelease&preserve-view=true#get_hiddenpdftoolbaritems "get_HiddenPdfToolbarItems - interface ICoreWebView2ExperimentalSettings6 | Microsoft Docs"
```